---
#full allowed collision matrix
moveit_msgs/AllowedCollisionMatrix allowed_collision_matrix
